### ISSUE TITLE
Getting the perk type of an armor perk

### ISSUE THREAD

**Issue by kjverble** [Bungie-net/api#845]
Hello,

I have code the gets the InventoryItemDefinition for an armor perk on an item. I can look at the itemTypeDisplayName and it shows 'Armor Perk'. That works fine. The problem is I can't see how to tell which set of armor perks it applies to. I want to differentiate between the perk set with the targeting and unflinching perks from the set with reserves and scavenger perks. What do I need to look at to tell which set the perk belongs to?

Thanks,
Kevin

**Comment by designker** [comment]
These arn't bugs but general help questions on the api. You might find the discord server at https://discord.gg/WJDSUgj  far better for these type of questions.

**Comment by kjverble** [comment]
Okay. Thanks. I'll try that.

**Comment by vthornheart-bng** [comment]
Sorry about the delay!  I totally missed this!  Did you get helped out on this one?

**Comment by kjverble** [comment]
No, but I don't think it matters much now. I changed my code to look at each bucket of perks separately. That doesn't tell me what socket set they belong to but it keeps them grouped separately which is all I care about right now.

**Comment by vthornheart-bng** [comment]
Good deal - hopefully when I get some time soon I'll be able to write up the much needed documentation about sockets - #735 - and when that happens there'll be a much better resource for this sort of thing.  Sockets started so simple, and became such a giant blob of all-encompassing features.
